ScreenControl_Web/src/Connection/MessageReceiver.ts

```diff
@@ -100,7 +111,37 @@ export class MessageReceiver {
                 if (key === _response.requestID) {
                     this.responseCallbacks[key].callback(_response);
                     delete this.responseCallbacks[key];
-                    break;
+                    return;
+                }
+            };
+        }
+
+        console.log("Received a WebSocketResponse that did not match a callback." +
+            "This is the content of the response: \n Response ID: " + _response.requestID +
+            "\n Original request - " + _response.originalRequest +
```

Comment:
Could consider rearranging this so the original request is the _last_ thing printed here because it's likely to be multiple lines long and interrupt the readability? Would need to do the same in the Unity client.